### DISCUSSION TITLE
How to find the layer group to which a certain two-dimensional material belongs?

### DISCUSSION THREAD

**Discussion by HangGao-ai** [spglib/spglib#513] [General]
Can I use spglib to obtain the layer group of a certain two-dimensional material? 
I tried using the ”get_layergroup“ function, but I still got its dataset, which does not include information about which of the 80 layer groups it belongs to.

**Comment by atztogo** [comment]
No. It is an experimental implementation, and we cannot support this function yet. We need to further improve the implementation and the design of the user interface.
